doc/integrator/authentication.rst

```diff
@@ -10,6 +10,22 @@ obtained from the "auth ticket" cookie set in the request.
 The policy is created, and added to the application's configuration, in the
 application's main ``__init__.py`` file.
 
+In the ``.env.sample`` we can congigure it with the following variable:
+
+.. code::
+
+   AUTHTKT_TIMEOUT  # Default to one day.
+   AUTHTKT_REISSUE_TIME  # Default to 2h30, good 10 times smaller than AUTHTKT_TIMEOUT.
```

Comment:
"good 10 times smaller than AUTHTKT_TIMEOUT."
-> "recommended to be 10 times smaller than AUTHTKT_TIMEOUT."